WL-300: Remove all usages of USE_CUSTOM_THEME and THEME_NAME from python files

saleem-latif opened the pull request:
Hi @mattdrayer , @asadiqbal08 

Kindly review this PR, it contains changes for [WL-300](https://openedx.atlassian.net/browse/WL-300)

__Description of [WL-300](https://openedx.atlassian.net/browse/WL-300):__
After we have completed comprehensive theme implementation for Stanford theme we would like to remove all instances of USE_CUSTOM_THEME and THEME_NAME variables that are no longer needed.

saleem-latif commented:
jenkins run python

mattdrayer commented:
This PR is looking good so far -- I know that test coverage is coming back as sufficient in terms of the changeset, but we should be sure we are properly covering template/theme-related workflows which touch this area to ensure there are no breakages.

saleem-latif commented:
@mattdrayer I have added more tests and also added placeholder templates that can be updated to achieve functionality same as was being achieved with `USE_CUSTOM_THEME`.

saleem-latif commented:
jenkins run js

saleem-latif commented:
jenkins run bokchoy

mattdrayer commented:
Overall I think we're in good shape with this PR now, although we should wait for #13039 to merge at this point.

mattdrayer commented:
👍 to merge after #13039 goes

saleem-latif commented:
going ahead with the merge now that https://github.com/edx/edx-platform/pull/13039 is merged.
